src/main/java/pl/javastart/task/loanProccess/Offer.java

```diff
@@ -0,0 +1,35 @@
+package pl.javastart.task.loanProccess;
+
+public class Offer {
+
+    private boolean valid;
+    private double value;
+    private double percentage;
+
+    Offer() {
+    }
+
+    boolean isValid() {
+        return valid;
+    }
+
+    void setValid(boolean valid) {
+        this.valid = valid;
+    }
+
+    double getValue() {
+        return value;
+    }
+
+    void setValue(double value) {
+        this.value = value;
+    }
+
+    double getPercentage() {
+        return percentage;
+    }
+
+    void setPercentage(double percentage) {
+        this.percentage = percentage;
+    }
```

Comment:
To jest ok, aczkolwiek jeżeli coś ograniczamy do konkretnego pakietu, to równie dobrze pola mogą być package private (domyślne) i wtedy gettery i settery nie są potrzebne